t/local_ingest.t

```diff
@@ -110,6 +110,35 @@ describe "HTFeed::PackageType::Simple" => sub {
       ok($testlog->matches(qr(File validation failed.*meta\.yml)s));
     }
   };
+
+  describe "HTFeed::PackageType::Simple::ImageRemediate" => sub {
+    it "compresses tif to a valid jpeg2000" => sub {
+      my $volume = unpacked_volume("rgb_tif");
+      my $remediate = HTFeed::PackageType::Simple::ImageRemediate->new(volume => $volume);
+      $remediate->run();
+
+      ok(-e "$tmpdirs->{ingest}/rgb_tif/00000001.jp2");
+      ok($remediate->succeeded());
+
+      HTFeed::PackageType::Simple::SourceMETS->new(volume => $volume)->run();
+
+      my $validate = HTFeed::VolumeValidator->new(volume => $volume);
+      $validate->run();
+      ok($validate->succeeded());
+    };
+
+    it "recompresses lossless jpeg2000 to a valid jpeg2000" => sub {
+      my $volume = unpacked_volume("lossless_jp2");
+
+      HTFeed::PackageType::Simple::ImageRemediate->new(volume => $volume)->run();
+      HTFeed::PackageType::Simple::SourceMETS->new(volume => $volume)->run();
+
+      my $validate = HTFeed::VolumeValidator->new(volume => $volume);
+      eval { $validate->run(); };
```

Comment:
probably don't need the `eval` here (since we would want the test to fail if `HTFeed::VolumeValidator` raises an exception)